core/src/datayoga_core/job.py

```diff
@@ -125,6 +125,49 @@ def transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> Job
 
         return result
 
+    async def async_transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> JobResult:
+        """
+        Transforms data(async version)
+
+        Args:
+            data (List[Dict[str, Any]]): Data
+            deepcopy: if True, performs a deepcopy before modifying records. otherwise, modifies in place. can affect performance.
```

Comment:
Change description to:
```
If set to True, performs a deepcopy before modifying records; otherwise, it modifies them in place. This can affect performance.
```

Reply:
✅